cypress/support/config/settings.js

```diff
@@ -4218,7 +4218,29 @@ module.exports = () => ({
         },
         smoke: false,
       },
-      onDemandTV: { environments: undefined, smoke: false },
+      onDemandTV: {
+        environments: {
+          live: {
+            paths: [
+              '/persian/bbc_persian_tv/tv_programmes/w13xttnr', // Brand
+              '/persian/bbc_persian_tv/tv/w172xbvktkvplq0', // Episode
+            ],
+            enabled: false,
+          },
+          test: {
+            paths: [
+              '/persian/bbc_persian_tv/tv_programmes/w13xttnr', // Brand
+              '/persian/bbc_persian_tv/tv/w172xbvktkvplq0', // Episode
+            ],
+            enabled: true,
+          },
+          local: {
+            paths: [],
+            enabled: true,
```

Comment:
What does local do if it's enabled but without any paths?

Reply:
Doesn't run any tests, but I haven't run the full suite on local for a while to check how it works in that situation. I have disabled it for now and will come back to this in my local config audit

Reply:
Nice one!

Reply:
If `enabled: true`, but path array is empty, it won't run any tests. So it doesn't really matter if it's disabled 😄